requirements.txt

```
@@ -1,8 +1,8 @@
-Sphinx==7.1.1
+Sphinx==7.2.3
```

```suggestion
Sphinx==7.2.4
```

---

I hope to release 7.2.5 today or tomorrow, so might wait to bump here.

---

Perhaps:

```suggestion
Sphinx~=7.2.4
```

which would capture further bumps in the 7.2 series (but not 7.3)